ufo/ufo-profiler.c

```diff
@@ -139,6 +142,14 @@ ufo_profiler_call (UfoProfiler    *profiler,
         cl_err = clEnqueueNDRangeKernel (command_queue, kernel, work_dim, NULL, global_work_size, local_work_size, 0, NULL, NULL);
     }
 
+    if (block) {
+        cl_err = clWaitForEvents (1, &event);
+        UFO_RESOURCES_CHECK_CLERR (cl_err);
+        if (!priv->trace) {
```

Comment:
I don't know what the tracing does with the event, so I release only if tracing is disabled. @matze please check.

Reply:
It keeps it alive for later analysis. You never used `ufo-launch -t ...` and then ran `ufo-prof opencl*.json`? But yes, if tracing is disabled you can safely release it.